code/modules/projectiles/updated_projectiles/guns/specialist.dm

```diff
@@ -203,22 +203,26 @@
 	wield_delay = 16
 	aim_slowdown = SLOWDOWN_ADS_SPECIALIST
 	var/datum/ammo/ammo_secondary = /datum/ammo/bullet/smartgun/lethal//Toggled ammo type
-	var/shells_fired_max = 20 //Smartgun only; once you fire # of shells, it will attempt to reload automatically. If you start the reload, the counter resets.
+	var/shells_fired_max = 50 //Smartgun only; once you fire # of shells, it will attempt to reload automatically. If you start the reload, the counter resets.
 	var/shells_fired_now = 0 //The actual counter used. shells_fired_max is what it is compared to.
 	var/restriction_toggled = 1 //Begin with the safety on.
 	gun_skill_category = GUN_SKILL_SMARTGUN
 	attachable_allowed = list(
+						/obj/item/attachable/extended_barrel,
 						/obj/item/attachable/heavy_barrel,
+						/obj/item/attachable/reddot,
+						/obj/item/attachable/flashlight,
```

Comment:
Some of these attachments are not much a case of trade-off options, but must-have instead. And smartgunners have night vision.
Also adding attachments because they look cool is nice and all, but try to get the sprite itself improved if that's what you want.

Reply:
More like yet another buff, and not a trade off option.

Reply:
It's fine. How many times have we done this, where you're worried about something being 'totally broken', I say its not, and it gets playtested and turns out to be fine?